app/views/events/transhipments/ContainerNumberView.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.{Button, ErrorSummary}
+@import views.html.components.InputText
+@import models.{MovementReferenceNumber, Index}
+@import controllers.events.transhipments.routes
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    button: Button,
+    inputText: InputText,
+    errorSummary: ErrorSummary
+)
+
+@(form: Form[_], mrn: MovementReferenceNumber, eventIndex: Index, containerIndex: Index, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = messages("containerNumber.title"),
+    showBackLink = true
```

Comment:
Also need `mrn = Some(mrn)` here. Pull main and check the other views for reference